workflow/rules/prepare.smk

```diff
@@ -70,48 +63,28 @@ rule finish_prepare:
 
 
 ###############################################################################
-### Download and process genome IDs
+### Trimm genome IDs
 ###############################################################################
-rule genome_process:
+rule trim_genome_seq_id:
     input:
-        script=os.path.join(config["scripts_dir"], "genome_process.sh"),
+        genome=config["genome_file"],
     output:
         genome=os.path.join(
             config["output_dir"], "genome.processed.fa"
         ),
```

Comment:
While you are at it: this looks like it fits in one row with less than 80 chars.